.generator/src/generator/openapi.py

```diff
@@ -569,3 +569,30 @@ def request(self):
         return Schema(
             next(iter(self.spec["requestBody"]["content"].values()))["schema"]
         )
+
+
+def get_default(operation, attribute_path):
+    attrs = attribute_path.split(".")
+    for name, parameter in parameters(operation):
+        if name == attrs[0]:
+            break
+    if name == attribute_path:
```

Comment:
This doesn't seem right to me. Is it meant to be nested in the for loop?

Reply:
No basically we're checking if we just try to get the default of a simple attribute.